doc/Makefile

```diff
@@ -53,8 +53,7 @@ clean:
 UPLOAD_DIR=/srv/docs_scipy_org/doc/scipy-$(RELEASE)
 
 SCIPYVER:=$(shell $(PYTHON) -c "import scipy; print(scipy.version.git_revision[:10])" 2>/dev/null)
-GITVER ?= $(shell cd ../tools; $(PYTHON) -c "from version_utils import git_version; \
-				print(git_version('..')[0][:10])")
+GITVER ?= $(shell (cd ..; set -o pipefail && git rev-parse HEAD 2>/dev/null | cut -c1-7) || echo Unknown)
```

Comment:
When I run `make dist` locally on this branch (I still use it to build docs for releases) I see `/bin/sh: 1: set: Illegal option -o pipefail` on this branch.

That said, `make dist` still succeeds in the end, and the `dist` build is also fine on this branch locally, so I'll go ahead and merge and just open a ticket for the `pipefail` thing just in case so I don't discover an issue during a rel process (if there is one).